### PR TITLE
Remove remains of the old bat gui tool

### PR DESCRIPTION
**Pull request by tuxmaster5000**
### Thank you for contributing to the Bareos Project!
I have found some rests of the old bar gui tool

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- ~~[ ] Your name is present in the AUTHORS file (optional)~~

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

##### General

- [x] PR name is meaningful
- [x] Purpose of the PR is understood
- ~~[ ] Separate commit for this PR in the CHANGELOG.md, PR number referenced is same~~
- [x] Commit descriptions are understandable and well formatted

##### Source code quality

- [x] `bareos-check-sources --since-merge` does not report any problems
- [x] `git status` should not report modifications in the source tree after building and testing

### CODE REVIEW

File: docs/manuals/images/image_scales [hunk 1]
```diff
@@ -5,12 +5,6 @@ back.png 0.65000
 bacula-applications.png 0.62500
 bacula-logo.png 0.65000
 bacula-objects.png 0.62500
-bat-brestore.png 0.19033
-bat-mediaview.png 0.32287
-bat11.png 0.34156
-bat12.png 0.34156
-bat13.png 0.31496
-bat_icon.png 0.65000
```
Comment (joergsteffens) [inline docs/manuals/images/image_scales:13]: I'm pretty sure, this file can be removed completely.

File: docs/manuals/source/Appendix/BareosPrograms.rst [hunk 2]
```diff
@@ -89,17 +89,6 @@ bareos-webui
...
-~~~
-
-:index:`\ <single: Command; bat>`\
-
-.. _bat:
-
-
-
-The Bacula/Bareos Administration Tool (:command:`bat`) has been a native GUI for Bareos. It has been marked deprecated since :sinceVersion:`15.2.0: bat vs. bareos-webui`. Since Bareos :sinceVersion:`17.2.0: bat: removed from core distribution` it is no longer part of Bareos. We encourage the use of |webui| instead.
-
```
Comment (joergsteffens) [inline docs/manuals/source/Appendix/BareosPrograms.rst:102]: While I understand the reasoning about removing this section, I'd prefer this section to stay. If somebody is wondering why he does not find bar, the documentation should show hin the reason.

**Review by joergsteffens — CHANGES_REQUESTED** [review]
Thank you for your contribution. You're absolutely right to remove references to bat. However, I'd prefer the documentation section to stay. Would you adapt your PR accordingly?

**Comment by tuxmaster5000** [comment]
I have done the modifications (hopefully) all. :)

**Comment by joergsteffens** [comment]
Oh, I fear there has been a mismatch of versions or branches. The second commit mess up some other sections. Either you can clean this or I'll take a look at this, probably next week.
One other thing: when this is done, I'd prefer a single commit (commit message of the first commit is fine).

**Comment by tuxmaster5000** [comment]
This have time, so you can do this next week. Its only an cosmetic "problem".

File: docs/manuals/images/image_scales (hunk 1 above)
Comment (joergsteffens) [reply docs/manuals/images/image_scales:13]: File removed.

File: docs/manuals/source/Appendix/BareosPrograms.rst (hunk 2 above)
Comment (joergsteffens) [reply docs/manuals/source/Appendix/BareosPrograms.rst]: Readded section

**Review by joergsteffens — APPROVED** [review]

**Pull request merged**